src/main/scala/diplomaticobjectmodel/logicaltree/LogicalTrees.scala

```diff
@@ -0,0 +1,61 @@
+// See LICENSE.SiFive for license details.
+
+package freechips.rocketchip.diplomaticobjectmodel.logicaltree
+
+import freechips.rocketchip.devices.debug._
+import freechips.rocketchip.devices.tilelink.{CLINT, TLPLIC}
+import freechips.rocketchip.diplomacy.{ResourceBindings, ResourceBindingsMap, SimpleDevice}
+import freechips.rocketchip.diplomaticobjectmodel.DiplomaticObjectModelAddressing
+import freechips.rocketchip.diplomaticobjectmodel.model._
+
+class CLINTLogicalTreeNode(device: SimpleDevice, f: () => OMRegisterMap) extends LogicalTreeNode {
```

Comment:
I think if you make this:
```
class CLINTLogicalTreeNode(device: SimpleDevice, f: => OMRegisterMap) extends LogicalTreeNode {
```
you can avoid having issues with lazy `module` evaluation without having to add in the the extra `CLINT.getOMRegMap` getter function

Reply:
I will check.

Reply:
val clintLogicalTree: CLINTLogicalTreeNode = new CLINTLogicalTreeNode(device, module.getOMRegMap)
 works